### ISSUE TITLE
No options besides removing it entirely?

### ISSUE THREAD

**Issue by Mane48** [ionuttbara/windows-defender-remover#132]
## *What* affects the bug ?
I dont know?

## *When* does this occur?
Every time I open it

## *In which* version of Windows does this issue happen ? Write version complete.
Windows 11 Home ( 10.0.22621 Build 22621 )

## *How* do we replicate the issue?
Run it?

## Expected behavior (if you have any solution write here.)
Youll get no options besides removing it!

## Other Comments
![image](https://github.com/ionuttbara/windows-defender-remover/assets/68643596/a50a512d-b103-42e5-aaee-fee6c03e7f56)
This is what I mean by no options, I dont know ifts meant to be like this I see other pictures with more options so im a bit confused!

**Comment by lightsabata** [comment]
I agree, i prefer the "safe methods" since "completley removing " option always makes new windows 11 machines that i install having a wierd behaviour or just became broken some times.

@ionuttbara is there à reason to remove the safe methods, i saw on the 1.6.9 release, you said that the project is ended.
So it is uesless to expect the return of the "safe method" on a update ?
Thanks for your hard work btw

**Comment by ionuttbara** [comment]
in Windows 11 2xH2 (2,3,4, etc) safe methods are patchable by microsoft and in some systems the *antimalware service executable* is not stoped as remover is done. 
I decided to pull out *safe methods* because its not working with newer Windows versions. If you have Windows 10 or earlier versions, you can download the 12.6.9 for that. 
This tool is focusing to remove Windows Defender entirely from system including components of Windows Security to prevent errors such as "AntiMalware Service/Engine not found" or something. Also values "DisableAntiSpyware"  is not working besides if you had disabled Tamper Protection.
So no more options for now. Maybe in next releases of Windows, but there are small chances to get that back.
